pymmcore_plus/_tests/test_core.py

```diff
@@ -19,11 +22,17 @@
     Metadata,
     PropertyType,
 )
+from pymmcore_plus.core._signals import _CMMCoreSignaler
 
 
 @pytest.fixture
 def core():
     core = CMMCorePlus()
+    # force using psygnal callbacks
+    # necessary until tests can be adapted to work using qtbot
+    core.events = _CMMCoreSignaler()
+    core._callback_relay = pymmcore_plus.core._mmcore_plus.MMCallbackRelay(core.events)
+    core.registerCallback(core._callback_relay)
```

Comment:
Otherwise I think there end up being some timing issues that require qtbot to resolve.  Ideally we can run tests with both types of callbacks, but until then this is maintaining the status quo of these tests (only using CMMCoreSignaler)